pkg/jx/cmd/add_app_test.go

```diff
@@ -188,20 +188,7 @@ func TestAddAppWithSecrets(t *testing.T) {
 		console.ExpectEOF()
 	}()
 
-	pegomock.When(testOptions.MockHelmer.UpgradeChart(
-		pegomock.AnyString(),
-		pegomock.EqString(name),
-		pegomock.AnyString(),
-		pegomock.EqString(version),
-		pegomock.AnyBool(),
-		pegomock.AnyInt(),
-		pegomock.AnyBool(),
-		pegomock.AnyBool(),
-		pegomock.AnyStringSlice(),
-		pegomock.AnyStringSlice(),
-		pegomock.EqString(kube.DefaultChartMuseumURL),
-		pegomock.AnyString(),
-		pegomock.AnyString())).
+	pegomock.When(testOptions.MockHelmer.UpgradeChart(pegomock.AnyString(), pegomock.EqString(name), pegomock.AnyString(), pegomock.EqString(version), pegomock.AnyBool(), pegomock.AnyInt(), pegomock.AnyBool(), pegomock.AnyBool(), pegomock.AnyStringSlice(), pegomock.AnyStringSlice(), pegomock.EqString(kube.DefaultChartMuseumURL), pegomock.AnyString(), pegomock.AnyString(), false)).
```

Comment:
sorry for the fomatting, goland did it :)

Reply:
You should be able to turn automatic wrapping of lines in Goland :-)

Reply:
it is already set (hard wrap at 120), looks like the refactor engine ignores this. will fix them up manually whilst fixing the tests